### ISSUE TITLE
Default SCHEMA per connection 

### ISSUE THREAD

**Issue by COBOL-Erik** [IBM/db2forzosdeveloperextension-about#137]
## Description of the feature or enhancement

I would like to be able to set default schemas on the connection level. I know you can set schema with the right-click options, but the default differs between connections, and I’d like to set it only once.

This request is somewhat similar to #60, but we really only have one default per connection.

**Comment by b-tsao** [comment]
You can add a default schema on the connection level by specifying the currentSchema connection property when add/editing a connection.

ref: https://ibm.github.io/db2forzosdeveloperextension-about/docs/the-basics/creating-a-database-connection

For additonal properties see JDBC properties:
https://www.ibm.com/docs/en/db2-for-zos/13?topic=pdsdjs-common-data-server-driver-jdbc-sqlj-properties-db2-db2-zos-servers

<img width="1402" alt="image" src="https://github.com/IBM/db2forzosdeveloperextension-about/assets/8595654/2379d14d-d42d-4134-a87d-b89a684280d5">

**Comment by kumari-mani** [comment]
Hi @COBOL-Erik  We have included this enhancement . As a result, the issue is now closed. Please feel free to open a new issue if you discover any.

**Comment by COBOL-Erik** [comment]
Perfect. Thank you.
